poc/idpf_poplar.sage

```diff
@@ -231,6 +233,23 @@ class IdpfPoplar(Idpf):
         return new_cls
 
 
+def pack_bits(bits: Vec[Field2]) -> Bytes:
```

Comment:
If my suggestion above is taken, then let's move these to common.sage.

Reply:
`unpack_bits()` uses the `Field2` constructor, so I can't directly move it to common.sage without hacks. field.sage already imports items from common.sage, so attempting to import Field2 at top level would create a circular import. Importing the field module at the top of the function would work, but I'd rather not do that. We could leave the functions where they are, or move them to field.sage, or make them deal in integers and move them to common.sage. I'll try out the last option.

Reply:
Good catch. I think we should keep them where they are then. I would still describe them in prose.